### ISSUE TITLE
How to check whether dependency exists

### ISSUE THREAD

**Issue by ernsheong** [di-ninja/di-ninja#7]
```js
this.di("i-dont-exist")
``` 

throws an error. Is there a gentler way to check whether a dependency exists?

**Comment by ernsheong** [comment]
Ah I found it: 
https://github.com/di-ninja/di-ninja/blob/138885ccb06131fb2f6deedbab68c8809c773179/src/makeContainerApi.js#L7

One for the docs perhaps? We should document entire `container` API here: https://github.com/di-ninja/di-ninja#5-container

**Comment by takion** [comment]
Yes you're right, I have not the time to do that for now, but feel free to make a pull request for this, I'll accept it.

PS: di-ninja remain maintained and a good choice in many situation but with the new stable react context api for the browser and the experimental feature [async_hook](https://nodejs.org/api/async_hooks.html) in node, I can do most of my dependency injection works natively.
